Norma-Portuguesa-405-citação-numérica

```diff
@@ -0,0 +1,522 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" default-locale="pt-PT">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Norma Portuguesa 405 (citação numérica)</title>
+    <title-short>NP 405</title-short>
+    <id>http://www.zotero.org/styles/norma-portuguesa-405-citacao-numerica</id>
+    <link href="http://www.zotero.org/styles/norma-portuguesa-405-citacao-numerica" rel="self"/>
+    <link href="https://github.com/citation-style-language/styles/pull/413" rel="documentation"/>
+    <author>
+      <name>SBIDM. Universidade de Aveiro</name>
```

Comment:
add yourself as author (not contributor), delete all other authors&contributors (relevant credit is given by the template link)